config.js

```diff
@@ -29,7 +29,7 @@ var config = {};
  * Default log level. Can be one of: 'DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL'
```

Comment:
One (or more) entries in CHANGES_NEXT_RELEASE should be added regarding this PR.